pkg/acceptance/acceptance.go

```diff
@@ -84,6 +84,34 @@ func RunAcceptance(ctx context.Context, logger kitlog.Logger) {
 					Expect(txact.Rollback()).To(Succeed(), "transaction to rollback, as connection should have been uninterrupted")
 				})
 			})
+
+			Context("With unhealthy keeper", func() {
+				var (
+					asyncKeeper string
+				)
+
+				BeforeEach(func() {
+					asyncKeeper = mustClusterdata(ctx, client).AsynchronousStandbys()[0].Spec.KeeperUID
+					Expect(
+						execCommand(ctx, "docker-compose", "exec", asyncKeeper, "pkill", "-STOP", "stolon-keeper"),
+					).To(Succeed(), "stopping %s stolon-keeper service", asyncKeeper)
+
+					Eventually(func() error { return mustClusterdata(ctx, client).CheckHealthy() }).Should(
+						Not(Succeed()), "timed out waiting for > min keepers to become unhealthy",
+					)
+				})
+
+				AfterEach(func() {
+					Expect(
+						execCommand(ctx, "docker-compose", "exec", asyncKeeper, "pkill", "-CONT", "stolon-keeper"),
+					).To(Succeed(), "starting %s stolon-keeper service", asyncKeeper)
+				})
+
+				It("Fails to failover without minimum number of healthy keepers", func() {
```

Comment:
This test name duplicates the context. Recall your test name is a concatenation of all your contexts, so this would read

```
With unheathly keeper
  It Fails to failover without minimum number of healthy keepers
```

You kinda repeat what has already been said by the context, which makes it a bit more difficult to parse

Reply:
Just cut it down to "fails to failover"?